src/app/component/eco-news/create-news/create-news-interface.ts

```diff
@@ -32,6 +32,7 @@ export interface NewsDTO {
   tags: Array<string>;
```

Comment:
Please, add description and specify bug's identifier

Reply:
fixed